kubernetes/linux/setup.sh

```diff
@@ -70,7 +70,7 @@ docker_cimprov_version=$(sudo tdnf list installed | grep docker-cimprov | awk '{
 echo "DOCKER_CIMPROV_VERSION=$docker_cimprov_version" >> packages_version.txt
 
 #install fluent-bit
-sudo tdnf install fluent-bit-2.1.10 -y
+sudo tdnf install fluent-bit-2.2.2 -y
```

Comment:
can you please double check these (1) verify if there are any errors/warnings in fluent-bit.log (2) did you verify with some workloads and see if fluent-bit able to detect the new log files and log updates (3) can you please review the release notes and issues in github and see if there are any issues reported on this version